.readthedocs.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+submodules:
+  include: all
+
+sphinx:
+    builder: html
+    configuration: docs/conf.py
+
+python:
+   version: 3.7
+   install:
+      - method: pip
+      - requirements: docs/requirements.txt
+        path: .
```

Comment:
Syntax error. RtD build should fail on this. `path` has to go together with `method`.